legacy/web/src/main/java/org/jboss/as/web/WebConnectorDefinition.java

```diff
@@ -203,7 +204,7 @@ private WebConnectorDefinition() {
                 WebExtension.getResourceDescriptionResolver(Constants.CONNECTOR),
                 new AddressToNameAddAdaptor(CONNECTOR_ATTRIBUTES),
                 CONNECTOR_ATTRIBUTES);
-                setDeprecated(WebExtension.DEPRECATED_SINCE);
+        this.setDeprecated(WebExtension.DEPRECATED_SINCE);
```

Comment:
If you convert this constructor to instead do super(new Parameters(...) you can configure all this stuff using its builder API, including the need for FAKE_UNDERTOW_LISTENER_CAPABILITY. With that you don't have to override registerCapabilities.

Doing it this way is fine though; this comment is more just an FYI.